tests/libmigration/BUILD.bazel

```diff
@@ -13,6 +13,7 @@ go_library(
         "//tests/flags:go_default_library",
         "//tests/framework/kubevirt:go_default_library",
         "//tests/framework/matcher:go_default_library",
+        "//tests/infrastructure:go_default_library",
```

Comment:
This dependency is bad. libmigration is an infra library. It should depend on tests packages.